pymc3/glm/linear.py

```diff
@@ -84,9 +84,10 @@ def __init__(self, x, y, intercept=True, labels=None,
 
     @classmethod
     def from_formula(cls, formula, data, priors=None, vars=None,
-                     name='', model=None, offset=0.):
+                     name='', model=None, offset=0., eval_env=0):
```

Comment:
This is great! I think both from formula methods need a docstring, because the `eval_env` kwarg is completely unexpected from the classes's doc.